core/vm/statedb_utils_test.go

```diff
@@ -0,0 +1,125 @@
+)
+
+func TestValidDeployerv2AfterHardFork(t *testing.T) {
+	statedb, _ := state.New(common.Hash{}, state.NewDatabase(rawdb.NewMemoryDatabase()), nil)
+
+	evm, err := newEVM(proxyAdminCaller, statedb)
+	if err != nil {
+		t.Fatal(err)
+	}
+	evm.Context.Time = uint64(time.Now().Unix())
+	_, logicContractWhitelist, _, err := evm.Create(AccountRef(proxyAdminCaller), common.FromHex(whitelistDeployerV2), math.MaxUint64/2, big0)
+	if (logicContractWhitelist == common.Address{}) {
+		t.Fatal("Failed For deploying logic Contract")
+	}
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	smcAbi, err := abi.JSON(strings.NewReader(proxyAbi))
+	if err != nil {
+		t.Fatal(err)
+	}
+	logicSmcAbi, err := abi.JSON(strings.NewReader(logicAbi))
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	initialisedData, _ := logicSmcAbi.Pack(initialize, adminCaller, []common.Address{common.BytesToAddress([]byte("sentry"))})
+	// "" will go constructor
```

Comment:
What does this mean?

Reply:
We need to call the constructor method, in the library, putting method "" will go to constructor logic.